File: exemplos/python/08_ogc_records.py
# Coleções do catálogo externo OGC API Records (GET /ogc/records/collections, escopo catalogo:ler)
from _apoio import chamar, exigir, exigir_chave

exigir_chave()
status, _cab, corpo = chamar("/ogc/records/collections")
exigir(status == 200, f"esperava 200 em /ogc/records/collections, veio {status}: {corpo}")
colecoes = corpo.get("collections", [])
print(f"{len(colecoes)} coleção(ões) publicada(s) em OGC API Records")
for c in colecoes:
    print(f"  {c.get('id')}: {c.get('title', '')}")

File: exemplos/js/08_ogc_records.mjs
// Coleções do catálogo externo OGC API Records (GET /ogc/records/collections, escopo catalogo:ler)
import { chamar, exigir, exigirChave } from "./_apoio.mjs";

exigirChave();
const { status, objeto } = await chamar("/ogc/records/collections");
exigir(status === 200, `esperava 200 em /ogc/records/collections, veio ${status}: ${JSON.stringify(objeto)}`);
const colecoes = objeto.collections ?? [];
console.log(`${colecoes.length} coleção(ões) publicada(s) em OGC API Records`);
for (const c of colecoes) console.log(`  ${c.id}: ${c.title ?? ""}`);
